### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by leormonteiro** [xcompact3d/Incompact3d#3]
Hello, 
what about to segregate .prm files inside a .prm folder?

**Comment by ptb0890** [comment]
I agree, also the BC-*.f90 files would help keep the code tidy I think.

**Comment by ricardofrantz** [comment]
How you like the idea of having a /src with the core code and leave the BC*.f90 and BC*.prm in the first level folder?

**Comment by ptb0890** [comment]
Sounds good to me. I haven't looked at the makefile so don't know how hard
it would be to rearrange things.

Paul

**Comment by mathrack** [comment]
For the periodic channel test case, we might consider using the data from Vreman and Kuerten as a reference instead of the one from Moser, Kim and Mansour? They are supposed to be more accurate, see [http://vremanresearch.nl/channel.html](http://vremanresearch.nl/channel.html)

**Comment by mathrack** [comment]
For the periodic channel test case, we might consider using an imposed pressure gradient instead of an imposed flow rate. The main advantage of the imposed pressure gradient is that the flow cannot remain laminar. In case of a laminar flow with an imposed pressure gradient, there is no friction at the wall, and the fluid will accelerate until transition to turbulence occurs.
